### PR TITLE
stars project0

### PR DESCRIPTION
**Pull request by AndriiKrytsia**
[DEMO LINK](https://AndriiKrytsia.github.io/layout_stars/)
- [TEST REPORT LINK](https://AndriiKrytsia.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--5 .stars__star:nth-child(-n+5) {
+  background-image: url(./images/star-active.svg);
+}
```
Comment (tetiana-ivanchenko) [inline src/style.css:36]: combine this selectors

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
+* {
+  margin: 0;
+}
```
Comment (tetiana-ivanchenko) [inline src/style.css:3]: you shouldn't reset indentations in the * selector

Comment (loralevitska) [reply src/style.css:3]: not fixed! Move this to body selector 
image

**Review by loralevitska — CHANGES_REQUESTED** [review]
Almost done) PLease fix previous comment

**Review by polosanya — APPROVED** [review]
Well done!

**Pull request closed without merging**